discovery-provider/src/tasks/generate_trending.py

```diff
@@ -197,7 +197,9 @@ def generate_trending(session, time, genre, limit, offset, strategy):
         if save_type == SaveType.track
     }
 
-    karma_query = get_karma(session, tuple(track_ids), None, False, xf)
+    karma_query = get_karma(
+        session, tuple(track_ids), strategy, None, False, xf, strategy
```

Comment:
The second param of `get_karma` seems to be `List[int]` but a Tuple is passed here?